README.md

```diff
@@ -41,8 +41,37 @@ TODO(https://github.com/mc-imperial/dredd/issues/29): provide support
 for Dredd releases (on Linux at least), so that folks can use Dredd
 without having to build it from source.
 
+
+We first show how to apply Dredd to a simple stand-alone program. We will then show how to apply it to a larger C++ CMake project.
+
+Before following these instructions, make sure you have built Dredd and that the `dredd` executable is located under `<repository-root>/third_party/clang+llvm-13.0.1/bin`.
+
+### Applying Dredd to a single file example
+
+`pi.cc` is a small example in `examples/simple` that calculates an approximation of pi. To run this example, run the following from the root of the repository:
+```
+# This will modify pi.cc in-place.
+third_party/clang+llvm-13.0.1/bin/dredd examples/simple/pi.cc
+# clang++ can be replaced with your favourite C++ compiler.
+clang++ examples/simple/pi.cc -o examples/simple/pi
+```
+Due to the changes made by Dredd, the compile command may lead to warnings being issued but this is expected.  The changes to `pi.cc` can be viewed by running `git diff` although, these changes are not meant to be human-readable.
+
+Running the executable with no additional arguments via `examples/simple/pi` will produce the expected result of 3.14159.  You can specify which mutants you want to use at runtime by running:
+```
+DREDD_ENABLED_MUTATION=$i$ examples/simple/pi
+```
+where $i$ is the id of the mutation in the program generated by Dredd. For example, running `DREDD_ENABLE_MUTATION=2 examples/simple/pi` will enable mutation number 2 in the modified program. 
+This may print a different output to the original program and in some cases may not terminate. 
+There is a possibility that equivalent mutants are generated which can lead to the printed value being unchanged.
+
+To restore the file `pi.cc` to it's initial state, run
+`git checkout HEAD examples/simple/pi.cc`
```

Comment:
Nit: I guess it would also be nice to show the `rm` command to get rid of the generated executable.